pkg/proxy/ipvs/proxier.go

```diff
@@ -562,21 +563,21 @@ func newServiceInfo(port *v1.ServicePort, service *v1.Service, baseInfo *proxy.B
 	return info
 }
 
-// KernelHandler can handle the current installed kernel modules.
+// KernelHandler can handle the current installed kernel modules for ipvs.
 type KernelHandler interface {
```

Comment:
I think we may want to split this into two separate interfaces at this point. `KernelHandler` and `ModulesHandler`.

Reply:
We would have to pass in `KerneHandler` into the constuctor for `ModulesHandler` since we compare the modules list based on the kernel version.

Reply:
That's good, but GetModules() is a complex implementation for ipvs, this pr is mainly to fix bugs. Can it to be done as optimize in another pr?